<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>401</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>470</width>
      <height>563</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QCheckBox" name="adRadioButton">
         <property name="text">
          <string>Modo Ambiente + Difusa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="adsRadioButton">
         <property name="text">
          <string>Modo Ambiente + Difusa + Especular</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Geral</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QDoubleSpinBox" name="iaDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Intensidade da Luz Ambiente</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QDoubleSpinBox" name="kaDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Coeficiente de Reflexão Ambiente</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QDoubleSpinBox" name="ilDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Intensidade da Luz Principal</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Coeficientes Difusos</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QDoubleSpinBox" name="kdSphereDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Coeficiente Difuso da Esfera</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QDoubleSpinBox" name="kdPlaneDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Coeficiente Difuso do Plano</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Constantes Especulares</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QDoubleSpinBox" name="kDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Constante de Ajuste de Distância</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QDoubleSpinBox" name="nDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Expoente de Brilho Especular</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Coeficientes Especulares</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QDoubleSpinBox" name="ksSphereDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Coeficiente Especular da Esfera</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QDoubleSpinBox" name="ksPlaneDoubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Coeficiente Especular do Plano</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="illuminateButton">
       <property name="text">
        <string>Aplicar Iluminação</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
